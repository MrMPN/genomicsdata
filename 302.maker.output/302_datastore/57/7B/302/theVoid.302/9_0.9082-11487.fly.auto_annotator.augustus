# This output was generated with AUGUSTUS (version 2.5.5).
# AUGUSTUS is a gene prediction tool for eukaryotes written by Mario Stanke (mario.stanke@uni-greifswald.de)
# and Oliver Keller (keller@cs.uni-goettingen.de).
# Please cite: Mario Stanke, Mark Diekhans, Robert Baertsch, David Haussler (2008),
# Using native and syntenically mapped cDNA alignments to improve de novo gene finding
# Bioinformatics 24: 637-644, doi 10.1093/bioinformatics/btn013
# reading in the file /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/9_0.9082-11487.fly.auto_annotator.xdef.augustus ...
# Setting 1group1gene for E.
# Sources of extrinsic information: M P E 
# Have extrinsic information about 1 sequences (in the specified range). 
# Initialising the parameters ...
# fly version. Use default transition matrix.
# Looks like /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/9_0.9082-11487.fly.auto_annotator.augustus.fasta is in fasta format.
# We have hints for 1 sequence and for 1 of the sequences in the input set.
#
# ----- prediction on sequence number 1 (length = 2405, name = 302) -----
#
# Constraints/Hints:
302	EST-EXON	exonpart	202	833	0	+	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-EXON	exonpart	860	1462	0	+	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-EXON	exonpart	1492	2190	0	+	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-INTRON	intronpart	1464	1490	0	+	.	grp=n_pieces;src=E "1;1;0:0"
302	PROTEIN	CDSpart	202	833	0	+	.	grp=p_pieces;src=P "100000;1;0:0"
302	PROTEIN	CDSpart	860	1462	0	+	.	grp=p_pieces;src=P "100000;1;0:0"
302	PROTEIN	CDSpart	1492	2190	0	+	.	grp=p_pieces;src=P "100000;1;0:0"
302	EST-EXON	genicpart	834	859	0	.	.	grp=n_peices;src=E "2;1;0:0"
302	EST-EXON	genicpart	1463	1491	0	.	.	grp=n_peices;src=E "2;1;0:0"
# Predicted genes for sequence number 1 on forward strand
# start gene g1
302	AUGUSTUS	gene	228	2205	0.4	+	.	g1
302	AUGUSTUS	transcript	228	2205	0.4	+	.	g1.t1
302	AUGUSTUS	start_codon	228	230	.	+	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	267	329	1	+	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	802	860	1	+	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	1086	1762	0.52	+	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	1965	2027	0.91	+	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	228	266	0.88	+	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	330	801	1	+	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	861	1085	0.78	+	2	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	1763	1964	0.95	+	2	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	2028	2205	0.85	+	1	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	stop_codon	2203	2205	.	+	0	transcript_id "g1.t1"; gene_id "g1";
# protein sequence = [MAENQQQQKKLLYLNPLTLEASFFLETLEEVNRQDQLQLDPHLTHLLERVVVGIETFLDKKPMYIVPKAAQQVSEEHP
# NTTDAGGITFVPPNELLKIKRTFNCTACTHRIVGTSINKAVAHLVEQHPNPNPNPHSTHNGNVNEQRPTTTKQEKKRAEVEARTKLNINLPKKVKALA
# LDDMKDLLRDKYQIADKLKVMPQYDEIEEDLCKLIAPAFAKETVRIYKFGSRITGIGTRSSDLDIFIDIAWITNFQQKSLSELGMQHIEVTVPILKEY
# LKDLLKFYATFNYERNLICPYFGKKHILISRIEKEMPLRYVDYANQNPECALQLRKPMVVQDPIQLNHNVTKAVTKYALQKFIDYCEQTATLLCSSI]
# Evidence for and against this transcript:
# % of transcript supported by hints (any source): 11.1
# CDS exons: 0/5
# CDS introns: 1/4
#      E:   1 
# 5'UTR exons and introns: 0/0
# 3'UTR exons and introns: 0/0
# hint groups fully obeyed: 1
#      E:   1 (n_pieces)
# incompatible hint groups: 2
#      E:   1 (n_peices)
#      P:   1 (p_pieces)
# end gene g1
###
# command line:
# /home/genomica/Programs/maker/exe/augustus/bin/augustus --species=fly --strand=forward --UTR=off --hintsfile=/home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/9_0.9082-11487.fly.auto_annotator.xdef.augustus --extrinsicCfgFile=/home/genomica/Programs/maker/exe/augustus/config/extrinsic/extrinsic.MPE.cfg /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/9_0.9082-11487.fly.auto_annotator.augustus.fasta
