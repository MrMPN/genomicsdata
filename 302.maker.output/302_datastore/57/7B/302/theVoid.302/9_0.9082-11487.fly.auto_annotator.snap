>302 9_0 offset:9082
Exon	168	266	+	25.276	0	0	2	302-snap.1
Exon	330	801	+	106.747	0	1	2	302-snap.1
Exon	861	1085	+	49.508	2	1	1	302-snap.1
Exon	1150	1256	+	24.484	2	0	2	302-snap.1
Exon	1316	1451	+	33.754	0	1	1	302-snap.1
Exon	1511	1687	+	48.212	2	1	0	302-snap.1
Exon	1763	1964	+	48.964	2	2	0	302-snap.1
Eterm	2028	2205	+	22.977	1	0	0	302-snap.1
